development/image-builder/main.go

```diff
@@ -209,6 +210,12 @@ func runBuildJob(o options, vs Variants, envs map[string]string) error {
 		buildArgs = envs
 	}
 
+	if len(o.buildArgs) > 0 {
+		for _, arg := range o.buildArgs {
+			buildArgs[arg.Name] = arg.Value
```

Comment:
As @dekiel requested, please also add a security gate that if argument is already set it skips the assignment to ensure variables set up previously without overriding them.
```suggestion
			if _, ok := buildArgs[arg.Name]; !ok {
			        buildArgs[arg.Name] = arg.Value
			}
```